api/data_explorer/controllers/export_url_controller.py

```diff
@@ -116,10 +116,6 @@ def _get_table_and_clause(es_field_name, field_type, value, bucket_interval,
     eg "((age76 >= 20 AND age76 < 30) OR (age76 >= 30 AND age76 < 40))".
     """
     sample_file_type_field = False
-    if field_type == 'samples_overview':
-        es_field_name = current_app.config['FACET_INFO']['Samples Overview'][
-            'elasticsearch_field_names'][value]
-        value = True
```

Comment:
Later on in this file:

```
        # facet_id is "Samples Overview" for the Samples Overview facet, and
        # es_field_name for all other facets.	        # es_field_name for all other facets.
```

Now that we no longer have "Samples Overview", get rid of `facet_id`. Just use `es_field_name`.

Reply:
Maybe I'm misunderstanding what you're saying here, but es_field_name isn't in this function. Are you suggesting to rename the variable facet_id to es_field_name?

Reply:
Yes. The only reason we called it `facet_id` instead of `es_field_name` was because of the Samples Overview facet.

Looks like you got most of the places, I think you missed a couple.